client/me/application-password-item/index.jsx

```diff
@@ -49,7 +49,7 @@ class ApplicationPasswordsItem extends React.Component {
 					<h2 className="application-password-item__name">{ password.name }</h2>
 					<p className="application-password-item__generated">
 						{ this.props.translate( 'Generated on %s', {
-							args: this.props.moment( password.generated ).format( 'MMM DD, YYYY @ h:mm a' ),
```

Comment:
What's actually the reason we use the @ here? According to http://momentjs.com/docs/#localized-formats `lll` would give us the date with time.

Reply:
I think it's just a stylistic preference, i.e `@` as a shorthand for "at". I guess that should actually be translatable, so maybe we should just switch to `lll`. What do you think?

Reply:
I think `lll` is preferable.